Makefile

```diff
@@ -15,4 +15,5 @@ install-python-dependencies:
 fetchschemas:
```

Comment:
Since this is now doing more than fetching -- and the fetching is really now serves secondary purpose -- should we rename it to "typecheck" or "mypy"?

Reply:
This is not really meant to typecheck. It does not show the output and it does not stop in case of conventional type errors. It runs mypy to ensure the schema is there and intact and that the processor references the right file. If the file is missing or in the wrong place mypy fatals, exit status 2, while conventional type errors are status 1. We fail only if mypy exit status is greater than 1.